### PR TITLE
add ZainMhrez.json

### PR DESCRIPTION
**Pull request by ZainMhrez**
add ZainMhrez.json

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1771"></a>

### CODE REVIEW

File: public/data/ZainMhrez.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/zainmhrez1",
+      "icon": "twitter"
+    },
+	{
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/zain-mhrez/",
+      "icon": "linkedin"
+    }
+  ],
```
Comment (Panquesito7) [inline public/data/ZainMhrez.json:22]: There's an extra comma in here.
```suggestion
  ]
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/ZainMhrez.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Zain Mhrez",
+  "type": "personal",
+  "bio": "Jr Frontend developer | Reactjs, learning typescript and web3",
+  "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
```
Comment (Panquesito7) [inline public/data/ZainMhrez.json:5]:
```suggestion
  "avatar": "https://github.com/https://github.com/ZainMhrez.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 🎉

**Pull request merged**